js/templates/pageNav.html

```diff
@@ -16,30 +16,30 @@
       </div>
       <div>
         <div class="flexVCent">
-          <a class="iconBtn ion-chevron-left js-navBack"></a>
+          <a class="iconBtn margLSm ion-chevron-left js-navBack"></a>
           <a class="iconBtn ion-chevron-right js-navFwd"></a>
-          <a class="iconBtn ion-refresh js-navReload"></a>
+          <a class="iconBtn ion-refresh js-navReload margRSm"></a>
```

Comment:
I'm not sure i'm following... I don't want extra spacing on the left and right side. I wanted a bit of extra space on the right side so that the refresh icon has some breathing room and isn't too close to the horizontal line. 

![screenshot 2017-01-31 13 16 15](https://cloud.githubusercontent.com/assets/13066886/22484719/8d49840e-e7b7-11e6-9ae3-69794079fc2b.png)

![screenshot 2017-01-31 13 17 57](https://cloud.githubusercontent.com/assets/13066886/22484758/b55356a0-e7b7-11e6-9884-5c7613bd1eb5.png)